openapi/docusign.monitor/openapi.yaml

```diff
@@ -0,0 +1,121 @@
+openapi: 3.0.1
+info:
+  title: Monitor API
+  description: DocuSign Monitor helps organizations protect their agreements with round-the-clock activity tracking. 
```

Comment:
Shall we follow the standard used in other connectors in the description as follows
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/ably/openapi.yaml#L14
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/spotify/openapi.yaml#L7